src/main.py

```diff
@@ -73,12 +74,14 @@ def main():
             style=style,
             history=history,
         )
-
-        command, args = handle_user_input(user_input, parser)
-        if command:
-            print(handler.execute(command, args))
+        if user_input:
+            command, args = handle_user_input(user_input, parser)
+            if command:
+                print(handler.execute(command, args))
             if command in (Menu.EXIT, Menu.CLOSE):
                 break
+        print()
```

Comment:
redundant

Reply:
no, it add space between results and new command line